boards/st/nucleo_l552ze_q/board.cmake

```diff
@@ -15,13 +15,11 @@ endif()
 
 # keep first
 board_runner_args(stm32cubeprogrammer "--port=swd" "--reset-mode=hw")
+include(${ZEPHYR_BASE}/boards/common/stm32cubeprogrammer.board.cmake)
 
 set_ifndef(BOARD_DEBUG_RUNNER pyocd)
 set_ifndef(BOARD_FLASH_RUNNER pyocd)
```

Comment:
I wonder if these 2 lines are still needed. They date from the very initial creation of this board support (6d31b1075af8c527d9e90ff89ee02d37ab03cc33). Now stm32cuberprogrammer is the default flash programming tool for all other ST boards.
@erwango, would you be ok these 2 lines are removed?